srcs/pressure_controller.cpp

```diff
@@ -305,20 +320,20 @@ void PressureController::compute(uint16_t p_centiSec) {
                                          m_pressure);
     }
 
-    DBG_PHASE_PRESSION(m_cycleNb, p_centiSec, 1u, m_phase, m_subPhase, m_pressure,
+    DBG_PHASE_PRESSION(m_cycleNb, p_tick, 1u, m_phase, m_subPhase, m_pressure,
                        m_blower_valve.command, m_blower_valve.position, m_patient_valve.command,
                        m_patient_valve.position)
 
 #if HARDWARE_VERSION == 2
-    m_alarmController->updateCoreData(p_centiSec, m_pressure, m_phase, m_subPhase, m_cycleNb);
-    sendDataSnapshot(p_centiSec, m_pressure, m_phase, m_subPhase, m_blower_valve.position,
-                     m_patient_valve.position, m_blower->getSpeed() / 10u, getBatteryLevel());
+    m_alarmController->updateCoreData(p_tick, m_pressure, m_phase, m_subPhase, m_cycleNb);
+    sendDataSnapshot(p_tick/2, m_pressure, m_phase, m_subPhase, m_blower_valve.position,
```

Comment:
What is the goal of this modification? `p_centisec` -> `p_tick/2`?

Reply:
it's a mistake.
fixed !